src/index.ts

```diff
@@ -4,6 +4,7 @@ export * from "./declaration";
 export * from "./error";
 export * from "./event-bridge";
 export * from "./expression";
+export { IIntegration as IntegrationHandler } from "./integration";
```

Comment:
This seems like a mistake?

Reply:
yes, lol, I hate that vscode tries to maintain old names when renaming...